### DISCUSSION TITLE
Search in files and ampersands: In the Find in Files result-panel ampersands are sometimes not shown but the next character is underlined

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#15] [1.2 Bugs and Issues]
Dear Alex, dear community,

I searched through a file in which I had copied the shortcuts, when I noticed something interesting: the ampersands are not shown. However, sometimes they lead to the letter afterwards to be underlined (like Backward, Forward, Keyword, Again).
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/937f29b2-3437-4498-88db-b51dd8681c24)
The settings that I used for the search:
<img width="307" alt="Pasted image 20240208195752" src="https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/9938a4b5-9066-4a83-ada6-c0ded9d2dd3b">
The behaviour seems to be different for more than one ampersand in the line, some of them not showing up:
<img width="186" alt="Pasted image 20240208200847" src="https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/f543d909-7c82-4527-8154-27267c1baddb">
WinEdt version: Build: 20230519  (v. 11.1) - 64-bit
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
This is correct. It turns out that custom-formatted text in Delphi's TListView control is by default formatted with ampersands interpreted as a prefix for hot keys (like the menu). The perceived inconsistencies are caused by the fact that only one such character is admissible in each line. Anyway, this has now been fixed by adding the [tfNoPrefix] option and ampersands are now treated as ordinary characters which is a desired behavior for all ListView controls in WinEdt.
